src/app/public/modules/confirm/confirm-config.ts

```diff
@@ -7,9 +7,28 @@ import {
 } from './confirm-button-config';
 
 export interface SkyConfirmConfig {
+  /**
+   * Specifies the message to display in a bold font at the top of the dialog.
```

Comment:
Can we change `a bold font` to just `bold`? I think that would be clearer.

Reply:
Done